notes.txt

```diff
@@ -0,0 +1,4 @@
+Roles: 0 - visitor/default
```

Comment:
```suggestion
Roles: 0 - User
1 - Admin
2 - Merchant
```

Reply:
We've since changed how these will be set up, but otherwise everything looks good here.